src/js/views/globals/prelogin/prelogin.css

```diff
@@ -102,7 +102,7 @@
 }
 
 
-.prelogin__continue-button .fa-sign-in-alt {
+.prelogin__continue-button .fa-right-to-bracket {
```

Comment:
We could keep this the same and it wouldn't modify any of the styling. But I figured we'd update it to match the new name Font Awesome gave the icon.

It coincides with the Handlebars template in the `src/js/views/globals/prelogin/login-prompt.hbs` file.